packages/amplify_datastore_plugin_interface/lib/src/types/models/uuid.dart

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import 'package:uuid/uuid.dart';
+
+class UUID {
+  static final _internal = Uuid();
+
+  static String getUUID() {
+    return _internal.v1();
```

Comment:
Android uses [V4 UUIDs](https://en.wikipedia.org/wiki/Universally_unique_identifier#Version_4_(random)). (See also, https://github.com/aws-amplify/amplify-android/issues/846.)

Reply:
updated to v4